pkg/fanal/analyzer/language/nodejs/yarn/testdata/yarn-classic-licenses/node_modules/is-callable/package.json

```diff
@@ -0,0 +1,106 @@
+{
+	"name": "is-callable",
+	"version": "1.2.7",
+	"author": {
+		"name": "Jordan Harband",
+		"email": "ljharb@gmail.com",
+		"url": "http://ljharb.codes"
+	},
+	"funding": {
+		"url": "https://github.com/sponsors/ljharb"
+	},
+	"contributors": [
+		{
+			"name": "Jordan Harband",
+			"email": "ljharb@gmail.com",
+			"url": "http://ljharb.codes"
+		}
+	],
+	"description": "Is this JS value callable? Works with Functions and GeneratorFunctions, despite ES6 @@toStringTag.",
+	"license": "MIT",
+	"main": "index.js",
+	"scripts": {
```

Comment:
I think we can remove the fields we don't need.
as for a `happy` test:
https://github.com/aquasecurity/trivy/blob/main/pkg/fanal/analyzer/language/nodejs/yarn/testdata/happy/package.json

Reply:
Yes, you're right

Reply:
done